allure-framework-modules/allure-selenium-module/src/main/java/com/capgemini/ntc/selenium/core/newDrivers/DriverExtention.java

```diff
@@ -337,6 +339,29 @@ public MenuElement elementMenu(By selector, By childsSelector, By subMenuSelecto
 		return new MenuElement(selector, childsSelector, subMenuSelector, childsSubMenuSelector);
 	}
 	
+	public HorizontalSliderElement elementHorizontalSlider(final By selector) {
+		BasePage.getAnalytics()
+						.sendMethodEvent(BasePage.analitycsCategoryName);
+		return new HorizontalSliderElement(selector);
+	}
+	
+	public HorizontalSliderElement elementHorizontalSlider(final By cssSelector, final By sliderSelector, final By valueSelector) {
+		BasePage.getAnalytics()
+						.sendMethodEvent(BasePage.analitycsCategoryName);
+		return new HorizontalSliderElement(cssSelector, sliderSelector, valueSelector);
+	}
+	
+	public HorizontalSliderElement elementHorizontalSlider(final By cssSelector,
```

Comment:
First parameter name needs to be more meaningful.

Reply:
Done.